planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/utils/calculation.cpp

```diff
@@ -339,10 +398,32 @@ double calc_lane_changing_acceleration(
     prepare_longitudinal_acc);
 }
 
+std::vector<double> calc_prepare_durations(const CommonDataPtr & common_data_ptr)
+{
+  const auto & lc_param_ptr = common_data_ptr->lc_param_ptr;
+  const auto threshold = common_data_ptr->bpp_param_ptr->base_link2front +
+                         lc_param_ptr->min_length_for_turn_signal_activation;
+  const auto max_prepare_duration = lc_param_ptr->lane_change_prepare_duration;
+
+  // TODO(Azu) this check seems to cause scenario failures.
+  if (common_data_ptr->transient_data.dist_to_terminal_start >= threshold) {
+    return {max_prepare_duration};
+  }
+
+  std::vector<double> prepare_durations;
+  constexpr double step = 0.5;
```

Comment:
This step is not a parameter ?

Reply:
originally it wasn't a parameter, so I kept it as it is.
@zulfaqar-azmi-t4 do you think there is a need to make it a parameter?